mwt-setup-scripts/configure-cluster-for-jenkins.sh

```diff
@@ -0,0 +1,366 @@
+#!/bin/bash
+
+set -x
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+#Basic cluster settings, change as needed.
+export DCOS_LOGIN_USERNAME=bootstrapuser;
+export DCOS_LOGIN_PASSWORD=deleteme;
+export CLUSTER_URL=$(dcos config show core.dcos_url);
+
+export JENKINS_STUB_URL='https://universe-converter.mesosphere.com/transform?url=https://infinity-artifacts.s3.amazonaws.com/permanent/jenkins/assets/scale-test-jenkins-2.150.1/stub-universe-jenkins.json';
+export EDGELB_STUB_URL='https://downloads.mesosphere.com/edgelb/v1.3.1/assets/stub-universe-edgelb.json';
+export EDGELB_POOL_STUB_URL='https://downloads.mesosphere.com/edgelb-pool/v1.3.1/assets/stub-universe-edgelb-pool.json';
+
+# Find security of the cluster
+SECURITY_MODE=`curl -k -H "Authorization: token=$(dcos config show core.dcos_acs_token)" $(dcos config show core.dcos_url)/dcos-metadata/bootstrap-config.json | jq .security | tr -d \"`
+
+install_mom()
+{
+    MOM_APP=marathon-1-8-${SECURITY_MODE}-mom-4.json
+    
+    # Add docker secret for pulling MoM-EE
+    dcos package install dcos-enterprise-cli --yes
+    dcos security secrets create -f docker/config.json docker-credentials
+
+    echo "Strict cluster detected, setting up security credentials!"
+
+    # auth for marathon to run as root and mom ees to run
+    dcos security org service-accounts keypair mom-private-key.pem mom-public-key.pem
+    dcos security org service-accounts create -p mom-public-key.pem -d "Marathon-EE service account" marathon_user_ee
+
+    curl -L -X PUT -k -H "Authorization: token=$(dcos config show core.dcos_acs_token)" $(dcos config show core.dcos_url)/acs/api/v1/acls/dcos:superuser/users/marathon_user_ee/full
+    curl -L -H 'Content-Type: application/json' -X PUT -k -H "Authorization: token=$(dcos config show core.dcos_acs_token)" -d '{"description": "dcos:mesos:master:task:user:root"}' $(dcos config show core.dcos_url)/acs/api/v1/acls/dcos:mesos:master:task:user:root
+    curl -L -X PUT -k -H "Authorization: token=$(dcos config show core.dcos_acs_token)" $(dcos config show core.dcos_url)/acs/api/v1/acls/dcos:mesos:master:task:user:root/users/dcos_marathon/create
+
+    dcos security secrets create-sa-secret --strict mom-private-key.pem marathon_user_ee service-credentials
+
+    echo "Installing MoM!"
+    dcos marathon app add $MOM_APP
+    echo "Done installing MoM!"
+}
+
+configure_jenkins()
+{
+    # Version of Jenkins deployed for MWT
+    dcos package repo add --index=0 jenkins-aws "${JENKINS_STUB_URL}";
+}
+
+install_edgelb()
+{
+    dcos package repo add --index=0 edgelb  "${EDGELB_STUB_URL}"
```

Comment:
I would recommend installing `edgelb-pool` stub here instead of in `install_edgelb_jenkins_pools`, it's an integral part of edge-lb and people may find it confusing that we install here only one.